docs/readmes/nms/alert.md

```diff
@@ -0,0 +1,322 @@
+---
+id: alerts
+title: Alerts
+hide_title: true
+---
+# Alerts
+
+Alerts are an important part of our NMS. We highly recommend the operators to run their networks with alerts always turned on. Without alerts, it is impossible to debug any potential issues happening on the network in a timely fashion.
+In this guide we will discuss
+
+* Viewing Alerts
+* Alert receiver configuration
+* Alert rules configuration
+    * Predefined alerts
+    * Custom Alerts
+* Troubleshooting
+
+## Viewing alerts
+
+### **Top Level Network Dashboard**
+
+Alert dashboard displays the current firing alerts in a table tabbed by severity. In each of the columns we additionally display the labels passed along with the alert.
+![viewing_alerts_1](assets/nms/userguide/alerts/viewing_alerts1.png)
+### **Alarm component’s Alert Tab**
+
+Alerts can also be viewed from the alert tab in the Alarm table.
+![viewing_alerts_2](assets/nms/userguide/alerts/viewing_alerts2.png)
+![viewing_alerts_3](assets/nms/userguide/alerts/viewing_alerts3.png)
+
+## Alert Receivers
+
+Alert Receiver is created to push the alert notification in real time so that the operator is notified instantaneously when the alert is fired. Following example details the steps involved in creating a slack alert receiver and configuring it in NMS.
```

Comment:
```suggestion
An alert receiver is created to push the alert notification in real time so that the operator is notified when the alert is fired. Following example details the steps involved in creating a slack alert receiver and configuring it in NMS.
```
it's not necessarily instantaneous, but just "real-time" so we don't want to create false expectations